<launch>
  <arg name="team" default="1"/>

  <node pkg="player" name="pointcloud_node" type="pointcloud_node" >
  	<param name="team_number" value="$( arg team )" />
  </node>

  <node pkg="player" name="map_node" type="map_node" output="screen">
  	<param name="team_number" value="$( arg team )" />
  </node>

  <node pkg="player" name="field_width_node" type="field_width_node" output="screen">
  	<param name="team_number" value="$( arg team )" />
  </node>


  <node pkg="player" name="location_node" type="location_node" output="screen">
	<param name="team_number" value="$( arg team )" />
  </node>

   
</launch>
